prisma/schema.prisma

```diff
@@ -4,8 +4,9 @@ generator client {
 }
 
 datasource db {
-  provider = "postgresql"
-  url      = env("DATABASE_URL")
+  provider          = "postgresql"
+  url               = env("DATABASE_URL")
+  shadowDatabaseUrl = env("SHADOW_DATABASE_URL")
```

Comment:
Not sure that is still required by default? Might by default always try to use this one, although not really required if the server supports creating new databases.

Reply:
Not sure why we need to add `shadowDatabaseUrl` here, I don't think it's mentioned in the Netlify guide

Reply:
Maybe because Heroku is used for the local database when using `migrate dev`.
But then it's also not mentioned in https://www.prisma.io/docs/guides/deployment/deployment-guides/deploying-to-heroku
So we could switch to `db push` and change later back to `migrate dev`